network/src/main/scala/com/linkedin/norbert/network/client/NetworkClient.scala

```diff
@@ -248,6 +352,49 @@ trait NetworkClient extends BaseNetworkClient {
     }
   }
 
+  /**
+   * Sends the alternative one way message to a node in the cluster. The <code>NetworkClient</code> defers to the current
+   * <code>LoadBalancer</code> to decide which <code>Node</code> the request should be sent to.
+   *
+   * @param request the message to send
+   *
+   * @throws InvalidClusterException thrown if the cluster is currently in an invalid state
+   * @throws NoNodesAvailableException thrown if the <code>LoadBalancer</code> was unable to provide a <code>Node</code>
+   * to send the request to
+   * @throws ClusterDisconnectedException thrown if the cluster is not connected when the method is called
+   */
+
+  def sendAltMessage[RequestMsg, ResponseMsg](request: RequestMsg)
+                                          (implicit is: InputSerializer[RequestMsg, ResponseMsg], os: OutputSerializer[RequestMsg, ResponseMsg]) {
+    doIfConnected {
```

Comment:
You don't need doIfConnected in this since the underlying function already has it.